draft-ietf-mls-protocol.md

```diff
@@ -2059,13 +2039,15 @@ A0  B0  C0  D0 -+- KD0
             |   +- ND1
             |
             D2 -+- KD2
```

Comment:
If KD2 and ND2 are on the diagram, I would prefer D3 to be there as well. The reader should see those as derived at the same time by consuming D2.

Reply:
As I said in the discussion on #146, this doesn't follow.  Just because you've generated KD2 and ND2 doesn't mean you have to derive forward to D3.  That's only required when you consume one of KD2/ND2.